src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -143,6 +143,7 @@ private void LoadParameters(ChainSpecJson chainSpecJson, ChainSpec chainSpec)
             Eip5656TransitionTimestamp = chainSpecJson.Params.Eip5656TransitionTimestamp,
             Eip6780TransitionTimestamp = chainSpecJson.Params.Eip6780TransitionTimestamp,
             Eip4788TransitionTimestamp = chainSpecJson.Params.Eip4788TransitionTimestamp,
+            Eip7516TransitionTimestamp = chainSpecJson.Params.Eip7516TransitionTimestamp,
```

Comment:
I still can't imagine the case when 4884 and 7516 have difference in activation. I'd not have a separate setting for it, simplifying all further specs and checks inside the code.

Reply:
I believe you can have 4844 without 7516, but not the opposite. that is why i added a check for that in the `ChainspecLoader`

Reply:
Yeah you can but what for? Do you expect any network to not active 7516?

Reply:
if core devs decided not to enable it for cancun but consider for a later fork, yes

Reply:
Not such impression I've got after today's ACDE. Don't forget about all the devops and users who will need to remember about this EIP in the future

Reply:
Do you think it is okay if we just by default include it with 4844 even if they kept them as separate PRs. if that is the case, i can remove this extra EIP number.